amenity-points.mss

```diff
@@ -4,1894 +4,1243 @@
+    shield-halo-fill: @standard-halo-fill;
+    shield-wrap-width: @standard-wrap-width;
+    shield-fill: @amenity-brown;
+    shield-text-dy: 11;
+    shield-unlock-image: true;
+
+    // second pass for icons without text where there is still space
+    a/marker-fill: @amenity-brown;
+    a/marker-placement: interior;
+    a/marker-clip: false;
+
+    [feature = 'tourism_alpine_hut'] {
+      shield-file: url('symbols/alpinehut.p.16.png');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/alpinehut.p.16.png');
+    }
     [feature = 'tourism_wilderness_hut'] {
-      marker-file: url('symbols/wilderness_hut.svg');
+      shield-file: url('symbols/wilderness_hut.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/wilderness_hut.svg');
+      a/marker-fill: @transportation-icon;
     }
-    marker-fill: @transportation-icon;
-    marker-placement: interior;
-    marker-clip: false;
-  }
+    [feature = 'amenity_shelter'] {
+      shield-file: url('symbols/shelter.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/shelter.svg');
+      a/marker-fill: @transportation-icon;
+    }
+    [feature = 'amenity_atm'] {
+      shield-file: url('symbols/atm.svg');
+      shield-name: "[operator]";
+      a/marker-file: url('symbols/atm.svg');
+    }
+    [feature = 'amenity_bank'] {
+      shield-file: url('symbols/bank.svg');
+      a/marker-file: url('symbols/bank.svg');
+    }
+    [feature = 'amenity_bar'] {
+      shield-file: url('symbols/bar.svg');
+      a/marker-file: url('symbols/bar.svg');
+    }
+    [feature = 'amenity_bicycle_rental'] {
+      shield-file: url('symbols/rental_bicycle.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/rental_bicycle.svg');
+      a/marker-fill: @transportation-icon;
+    }
+    [feature = 'amenity_taxi'] {
+      shield-file: url('symbols/taxi.svg');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/taxi.svg');
+      a/marker-fill: @transportation-icon;
+    }
+    [feature = 'amenity_bus_station'] {
+      shield-file: url('symbols/bus_station.n.16.png');
+      shield-fill: @transportation-text;
+      a/marker-file: url('symbols/bus_station.n.16.png');
+    }
+    [feature = 'amenity_cafe'] {
+      shield-file: url('symbols/cafe.svg');
+      a/marker-file: url('symbols/cafe.svg');
+    }
+    [feature = 'tourism_artwork'] {
+      shield-file: url('symbols/artwork.svg');
+      a/marker-file: url('symbols/artwork.svg');
+    }
+    [feature = 'amenity_cinema'] {
+      shield-file: url('symbols/cinema.svg');
+      a/marker-file: url('symbols/cinema.svg');
+    }
+    [feature = 'highway_bus_stop'] {
+      shield-file: url('symbols/bus_stop.12.svg');
+      shield-fill: @transportation-text;
+      shield-text-dy: 9;
+      a/marker-file: url('symbols/bus_stop.12.svg');
+      a/marker-fill: @transportation-icon;
```

Comment:
Strictly speaking we don't but I wanted to keep those as we did have them before. But when I think about it it is redundant and probably should be removed. Should I?